db/migrate/20160414185708_create_container_label_tag_mappings.rb

```diff
@@ -0,0 +1,13 @@
+class CreateContainerLabelTagMappings < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_label_tag_mappings do |t|
+      t.string :labeled_resource_type
```

Comment:
I try to avoid `_type` column names to avoid confusion, but there's nothing wrong with it.  I usually take a hard stance against `_id` named columns though.